os/net/lwip/src/core/ipv4/igmp.c

```diff
@@ -111,7 +111,7 @@ Steve Reynolds
 #include <net/lwip/stats.h>
 
 #include <string.h>
-
+#include <stdlib.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding -Werror flag we were geting error "implicit declaration of function 'rand' [-Werror=implicit-function-declaration]"   . To overcome this error we added stdlib.h